admin/resources/js/components/PoolDashboard.tsx

```diff
@@ -54,7 +56,28 @@ const routes: Routes<RouterResult> = [
   {
     path: "/classifications",
     action: () => ({
-      component: <ClassificationTableApi />,
+      component: (
+        <div>
+          <Link href="/classifications/create" title="">
+            Create Classification
+          </Link>
+          <ClassificationTableApi />
```

Comment:
I noticed you had added to storybook, but is there any reason you didn't add the **Edit** column to the `ClassificationTableApi` component so classifications can be edited for real?